submissions/onyemesimChidiebereSabinus/html/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My e-Commerce site</title>
+</head>
+<body>
+
+  <H1> LOUIS     U.S     USED    CAR     SALES</H1>
+  <H4>...<em>authorised to explore</em></H4>
+    ...
+    <h1><em>We hold you by the hand and lead you to purchase and manage
+         your american used car at the best price you 
+    ,count on us, Do the ordering and leave the rest to us</em>
+    </h1>
+                 <img src="https://ford-h.assetsadobe.com/is/image/content/dam/vdm_ford/live/en_us/ford/nameplate/mustang/2020/collections/dm/20_FRD_MST_40830.tif?croppathe=1_3x2&wid=1440">
+                 <br>
+                 <br>
+
+                 <h2>Name:     MUSTANG GTRE00</h2>    <br>
+                 <h2>Price:                                      $290</h2><br>
+                
+                  <h1> <a href="#"><em>place your order now</em></a></h1>
+    <h2>PRODUCT DESCRIPTION</h2>
+    <h3>Unparalleled performance.
+        Six available high-powered engines means that every 2017 Mustang comes with performance in its DNA. 
+        And with nine unique trims, you’ve got options, from the fun and energetic GT Convertible to the 2017 Shelby GT500®,
+         host to the most power-dense supercharged V8 production engine in the world.
+        </h3>
+
+        <img src="https://s.aolcdn.com/dims-global/dims3/GLOB/legacy_thumbnail/788x525/quality/85/https://s.aolcdn.com/commerce/autodata/images/USC90CAS011B021001.jpg">
+        <h2>Name:  ESCALADE</h2>
+        <br>
+        <h2>Price:     $4555</h2>
+
+       <h1> <a href="#"><em>place your order now</em></a></h1>
+        <br>
+        <h2>PRODUCT DESCRIPTION</h2>
+        The 2020 Cadillac Escalade boasts a strong engine and high tow ratings. However,
+         it ranks in the bottom half of the luxury large SUV class because it struggles 
+         to match the interior opulence and ride quality of its rivals.
+        
+        
+        <br>
+    
+    <br>
+    <h1>Other similar products avalaible</h1>
+    <ol>
+        <li>Renault</li>
+        <li>Dungeon Ram</li>
+        <li>Santiago e-classic</li>
+        <li>Camry</li>
+        <li>Tesla</li>
+
+
+
+    </ol>
+    <h6>Prices are <span>negotiable</span> for customers from Nigeria only</h6>
```

Comment:
Why is `negotiable` in a `span` tag. I don't think it needed. You can remove it.